systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -1503,42 +1495,19 @@ void testOwnerReferenceOfCASecrets(ExtensionContext extensionContext) {
 
     @ParallelNamespaceTest
     void testClusterCACertRenew(ExtensionContext extensionContext) {
-        checkClusterCACertRenew(extensionContext, false);
-    }
-
-    @ParallelNamespaceTest
-    void testCustomClusterCACertRenew(ExtensionContext extensionContext) {
-        checkClusterCACertRenew(extensionContext, true);
-    }
-
-    void checkClusterCACertRenew(ExtensionContext extensionContext, boolean customCA) {
         final String namespaceName = StUtils.getNamespaceBasedOnRbac(namespace, extensionContext);
         final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
         final LabelSelector kafkaSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.kafkaStatefulSetName(clusterName));
         final LabelSelector zkSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.zookeeperStatefulSetName(clusterName));
 
-        if (customCA) {
-            generateAndDeployCustomStrimziCA(namespaceName, clusterName);
-            checkCustomCAsCorrectness(namespaceName, clusterName);
-            resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName, 3)
-                .editOrNewSpec()
-                    .withNewClusterCa()
-                        .withRenewalDays(15)
-                        .withValidityDays(20)
-                        .withGenerateCertificateAuthority(false)
-                    .endClusterCa()
-                .endSpec()
-                .build());
-        } else {
-            resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName, 3)
-                .editOrNewSpec()
-                    .withNewClusterCa()
-                        .withRenewalDays(15)
-                        .withValidityDays(20)
-                    .endClusterCa()
-                .endSpec()
-                .build());
-        }
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName, 3)
```

Comment:
I assume this change is part of moving the custom CA stuff to separate class?

Reply:
Yes, correct :)